src/serlo-editor/plugins/multimedia/toolbar/toolbar-wrapper.tsx

```diff
@@ -0,0 +1,51 @@
+import { faCog } from '@fortawesome/free-solid-svg-icons'
+import { ReactNode, useState } from 'react'
+
+import { FaIcon } from '@/components/fa-icon'
+import { ModalWithCloseButton } from '@/components/modal-with-close-button'
+import { useEditorStrings } from '@/contexts/logged-in-data-context'
+import { PluginToolbar } from '@/serlo-editor/editor-ui/plugin-toolbar'
+import { PluginDefaultTools } from '@/serlo-editor/editor-ui/plugin-toolbar/plugin-tool-menu/plugin-default-tools'
+import { EditorPluginType } from '@/serlo-editor-integration/types/editor-plugin-type'
+
+interface Props {
+  id: string
+  children: ReactNode
+  strings: ReturnType<typeof useEditorStrings>
```

Comment:
is there an upside to passing it on?
I usually prefer to use the hook again in the child component so we don't have to pass too much stuff down but that might also be a matter of taste.